src/main/java/org/radarcns/webapp/StatusEndPoint.java

```diff
@@ -0,0 +1,101 @@
+package org.radarcns.webapp;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.radarcns.auth.exception.NotAuthorizedException;
+import org.radarcns.security.exception.AccessDeniedException;
+import org.radarcns.webapp.util.CSVData;
+import org.radarcns.webapp.util.ResponseHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.radarcns.auth.authorization.Permission.MEASUREMENT_READ;
+import static org.radarcns.auth.authorization.RadarAuthorization.checkPermission;
+import static org.radarcns.security.utils.SecurityUtils.getJWT;
+
+@Path("/status")
+public class StatusEndPoint {
+
+    private static final String CSV_FILE_PATH = "/usr/local/tomcat/bin/radar/bins.csv";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StatusEndPoint.class);
+
+    @Context private HttpServletRequest request;
+
+    @GET
+    @Produces("text/csv")
```

Comment:
How is this to be used? Most REST clients do not natively understand CSV, so outputting JSON here might be a better fit. Querying this per topic/source/time might also be good, depending on how this is to be used. Or aggregating the data so that clients do not need to get everything but only `status/hdfs`:
```json
{
  "topics": {
     "android_phone_contacts": "healthy"
   }
}
```
or more verbose
```json
{
  "topics": {
     "android_phone_contacts": {
        "status": "healthy",
        "sources": {
            "abcd": {"status": "healthy", "lastUpdate": "2017-12-18T12:00:00Z", "count": 10, "total": 12010}
        }
     }
   }
}
```